modules/checking-mco-node-status.adoc

```diff
@@ -0,0 +1,96 @@
+ip-10-0-12-194.ec2.internal   True      False             False              False                    False              False
+ip-10-0-17-102.ec2.internal   False     True              False              False                    False              False
+ip-10-0-2-232.ec2.internal    False     False             True               False                    False              False
+ip-10-0-59-251.ec2.internal   False     False             False              True                     False              False
+ip-10-0-59-56.ec2.internal    False     False             False              False                    True               True
+ip-10-0-6-214.ec2.internal    False     False             Unknown            False                    False              False
+----
+
+where:
+
+UPDATED:: The `True` status indicates that the MCO has applied the current machine config to that particular node. The `False` status indicates that the node is currently updating.
+UPDATEPREPARED:: The `False` status indicates that the MCO has not started reconciling the new machine configs to be distributed. The `True` status indicates that the MCO has completed this phase of the update.
+UPDATEEXECUTED:: The `False` status indicates that the MCO has not started cordoning and draining the node. It also indicates that the disk state and operating system have not started updating. The `True` status indicates that the MCO has completed this phase of the update.
+UPDATEPOSTACTIONCOMPLETED:: The `False` status indicates that the MCO has not started rebooting the node or closing the daemon. The `True` status indicates that the MCO has completed reboot and updating the node status. The `Unknown` status indicates either that an error has occurred during the update process at this phase, or that the MCO is currently applying the update.
+UPDATECOMPLETED:: The `False` status indicates that the MCO has not started uncordoning the node and updating the node state and metrics. The `True` status indicates that the MCO has finished updating the node state and available metrics.
+RESUMED:: The `False` status indicates that the MCO has not started the config drift monitor. The `True` status indicates that the node has resumed operation.
+
+[NOTE]
+====
+Within the primary phases described above, there can be zero or more secondary phases that you can use to see upgrade progression. You can get more information that includes secondary phases of updates by using the `-o wide` option of the preceding command. This provides the additional `UPDATECOMPATIBLE`, `UPDATEFILESANDOS`, `DRAINEDNODE`, `CORDONEDNODE`, `REBOOTNODE`, `RELOADEDCRIO` and `UNCORDONED` columns. These secondary phases do not always occur and depend on the type of update you want to apply.
+====
+
+. Check the update status of an individual node by running the following command:
++
+[source,terminal]
+----
+$ oc describe machineconfignode/ip-10-0-12-194.ec2.internal <1>
```

Comment:
the MCN object name is the node's name. might be a good thing to note that users can depend on that even if they nodes aren't the standard *.internal naming scheme.